### PR TITLE
[#4678] improve(auth-ranger): Shrink Ranger authorization plugin package size

### PR DESCRIPTION
**Pull request by xunliu**
### What changes were proposed in this pull request?

Exclude invalidate depends on JAR package in the Rangere authorization module.

```
-rw-r--r--  1 xun  staff   657952 Aug 29 11:27 commons-lang3-3.14.0.jar
-rw-r--r--  1 xun  staff   238400 Aug 29 11:27 commons-text-1.10.0.jar
-rw-r--r--  1 xun  staff  1621691 Aug 29 11:27 jersey-bundle-1.19.3.jar
-rw-r--r--  1 xun  staff   780664 Aug 29 11:27 jackson-mapper-asl-1.9.13.jar
-rw-r--r--  1 xun  staff    19936 Aug 29 11:27 jsr305-3.0.2.jar
-rw-r--r--  1 xun  staff   284220 Aug 29 11:27 commons-lang-2.6.jar
-rw-r--r--  1 xun  staff     4112 Aug 29 11:27 gethostname4j-1.0.0.jar
-rw-r--r--  1 xun  staff   126898 Aug 29 11:27 javax.ws.rs-api-2.1.1.jar
-rw-r--r--  1 xun  staff    16830 Aug 29 11:27 error_prone_annotations-2.21.1.jar
-rw-r--r--  1 xun  staff     2199 Aug 29 11:27 listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar
-rw-r--r--  1 xun  staff   232248 Aug 29 11:27 jackson-core-asl-1.9.13.jar
-rw-r--r--  1 xun  staff   224460 Aug 29 11:27 checker-qual-3.37.0.jar
-rw-r--r--  1 xun  staff    34760 Aug 29 11:27 gravitino-authorization-ranger-0.7.0-incubating-SNAPSHOT.jar
-rw-r--r--  1 xun  staff   128076 Aug 29 11:27 jaxb-api-2.3.1.jar
-rw-r--r--  1 xun  staff  1055245 Aug 29 11:27 ranger-plugins-common-2.4.0.jar
-rw-r--r--  1 xun  staff     4617 Aug 29 11:27 failureaccess-1.0.1.jar
-rw-r--r--  1 xun  staff    30299 Aug 29 11:27 ranger-intg-2.4.0.jar
-rw-r--r--  1 xun  staff   700226 Aug 29 11:27 gravitino-core-0.7.0-incubating-SNAPSHOT.jar
-rw-r--r--  1 xun  staff    18336 Aug 29 11:27 jackson-jaxrs-1.9.13.jar
-rw-r--r--  1 xun  staff   632505 Aug 29 11:27 commons-configuration2-2.8.0.jar
-rw-r--r--  1 xun  staff    61829 Aug 29 11:27 commons-logging-1.2.jar
-rw-r--r--  1 xun  staff   229171 Aug 29 11:27 gravitino-api-0.7.0-incubating-SNAPSHOT.jar
```

### Why are the changes needed?

Reduce Rangere authorization library package size from `16.7 MB` to `7.1 MB`

Fix: #4678

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

CI passed.

### CODE REVIEW

File: authorizations/authorization-ranger/build.gradle.kts [hunk 1]
```diff
@@ -32,13 +32,10 @@ dependencies {
     exclude(group = "*")
   }
   implementation(libs.bundles.log4j)
-  implementation(libs.commons.collections4)
   implementation(libs.commons.lang3)
   implementation(libs.guava)
```
Comment (jerryshao) [inline authorizations/authorization-ranger/build.gradle.kts:36]: I think the jars like log4j, guava are already provided by Gravitino's base classpath, so we can remove these when doing copyLib, you can refer to @yuqi1129 's PR #4521

**Comment by jerryshao** [comment]
LGTM, @yuqi1129 you wanner take another look?

**Comment by yuqi1129** [comment]
@xunliu 
Do we need the package `jersey-bundle-1.19.3.jar` for Ranger?

**Comment by xunliu** [comment]
> @xunliu Do we need the package `jersey-bundle-1.19.3.jar` for Ranger?

Yes, We need it, Because `RangerClientExtend.java` depends on it.

**Review by yuqi1129 — APPROVED** [review]

**Pull request merged**